### DISCUSSION TITLE
Cassandra Filer store missing rows

### DISCUSSION THREAD

**Discussion by MatinZare** [seaweedfs/seaweedfs#6127] [General]
hi, i have a instance with on central scylla db as filer store. some rows are missing and for some files, I am getting 404. due to some data center issue it shutdowns ungracefully once, what is your solution and changes for the future so this doesn't happen? what about switching to PostgreSQL? it's not better?

**Comment by chrislusf** [comment]
Maybe. Need to know the reason first. Is it similar to https://github.com/seaweedfs/seaweedfs/issues/6113 ?
